apps/explorer/src/pages/transaction-result/programmable-transaction-view/TransactionsCard.tsx

```diff
@@ -0,0 +1,50 @@
+// Copyright (c) Mysten Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+import { type SuiTransaction } from '@mysten/sui.js';
+
+import { Transaction } from './Transaction';
+
+import { ExpandableList, ExpandableListItems } from '~/ui/ExpandableList';
+import { TransactionCard, TransactionCardSection } from '~/ui/TransactionCard';
+
+interface TransactionsCardProps {
+    transactions: SuiTransaction[];
+}
+
+export function TransactionsCard({ transactions }: TransactionsCardProps) {
+    const collapsedThreshold = transactions.length > 5;
+    const defaultItemsToShow = collapsedThreshold ? 5 : transactions?.length;
+
+    if (!transactions?.length) {
+        return null;
+    }
+
+    const expandableItems = transactions.map((transaction, index) => {
+        const [[type, data]] = Object.entries(transaction);
+
+        return (
+            <TransactionCardSection
+                key={index}
+                title={type}
+                collapsedOnLoad={collapsedThreshold}
+            >
+                <Transaction key={index} type={type} data={data} />
+            </TransactionCardSection>
+        );
+    });
+
+    return (
+        <TransactionCard collapsible title="Transactions">
+            <ExpandableList
+                items={expandableItems}
+                defaultItemsToShow={defaultItemsToShow}
+                itemsLabel="Inputs"
```

Comment:
This label seems wrong?

Reply:
What should we call this? Figma says `Inputs` and on current version we also call that section `Inputs`